### PR TITLE
bug(md-slide-toggle): slide toggle should fit to it's content width

### LINKED ISSUE #3230: bug(md-slide-toggle): selectable area gets 100% of container width
#### Bug, feature request, or proposal:
bug

#### What is the expected behavior?
The selectable area from md-slide-toggle should fit to it's content

#### What is the current behavior?
md-slide-toggle gets 100% of the container width as selectable area

#### What are the steps to reproduce?
[Plunker](http://plnkr.co/edit/PF5nzeUhUbtCBbsw2utR?p=preview)

#### Which versions of Angular, Material, OS, browsers are affected?
"@angular/material" :"2.0.0-beta.2"

### PR DESCRIPTION
**Pull request by jefersonestevo**
Fixes #3230

### CODE REVIEW

**Comment by DevVersion** [comment]
Please update the demo-app to have each `md-slide-toggle` element in its own row.

Also we should ensure that the baseline still works properly for the slide-toggle. See the `baseline/` route.

**Comment by jefersonestevo** [comment]
@DevVersion I've updated the demo app to show each slide-toggle on it's own row, but there are no slide-toggle on the baseline route. Should I add it there?

**Comment by jefersonestevo** [comment]
@DevVersion is there anything else that needs to change here?

**Comment by DevVersion** [comment]
@jefersonestevo Sorry for the delay. I will look into it tomorrow!

File: src/demo-app/slide-toggle/slide-toggle-demo.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.slide-toggle-example {
+  display:flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-content: flex-start;
+  align-items: flex-start;
```
Comment (DevVersion) [inline src/demo-app/slide-toggle/slide-toggle-demo.scss]: Only doing the following is enough:

```scss
display: flex;
flex-direction: column;
```

File: src/demo-app/slide-toggle/slide-toggle-demo.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.slide-toggle-example {
+  display:flex;
```
Comment (jefersonestevo) [inline src/demo-app/slide-toggle/slide-toggle-demo.scss]: Done

Comment (DevVersion) [inline src/demo-app/slide-toggle/slide-toggle-demo.scss]: Cool thx. Why still using `align-items`?

File: src/demo-app/slide-toggle/slide-toggle-demo.scss (hunk 1 above)
Comment (jefersonestevo) [inline src/demo-app/slide-toggle/slide-toggle-demo.scss]: @DevVersion although just these properties set each slide toggle on it's own row, I kept the property `align-items: flex-start;` so the demo will not stretch the lines

**Comment by DevVersion** [comment]
Thanks for the PR. I think we should consider this as a breaking change? @jelbourn

File: src/demo-app/slide-toggle/slide-toggle-demo.scss (hunk 2 above)
Comment (jefersonestevo) [inline src/demo-app/slide-toggle/slide-toggle-demo.scss]: Just to show that the slide toggle now has a display inline-block. If I do not use it, the browser will stretch the line and you can still click on the whole line to check the slide toggle.

[Example](http://plnkr.co/edit/iuTIRXgSXeQFWPopAgAA?p=preview)

Comment (DevVersion) [inline src/demo-app/slide-toggle/slide-toggle-demo.scss]: Thanks. Yeah I already tested it locally. Looks good!

**Comment by jelbourn** [comment]
In the issue I don't quite follow what you mean by "selectable area"- could you elaborate?

**Comment by jefersonestevo** [comment]
@jelbourn if you look in this [Plunker](http://plnkr.co/edit/iNnkg4bnxJm4YOx93wnD?p=preview) you can see that, anywhere in the gray area that you click, you'll change the value from the slide toggle (it uses all the available width from it's parent container). The same doesn't happen with the md-checkbox, that fit's just to it's content width.

**Pull request merged**